handbook/company/communications.md

```diff
@@ -493,7 +493,16 @@ You can learn more about how Fleet approaches security in the [security handbook
 
 
 ## Vendor questionnaires 
-In responding to security questionnaires, Fleet endeavors to provide full transparency via our [security policies](https://fleetdm.com/handbook/security/security-policies#security-policies) and [application security](https://fleetdm.com/handbook/business-operations/application-security) documentation. In addition to this documentation, please refer to [the vendor questionnaires page](https://fleetdm.com/handbook/business-operations/vendor-questionnaires) 
+In responding to security questionnaires, Fleet endeavors to provide full transparency via our [security policies](https://fleetdm.com/handbook/security/security-policies#security-policies) and [application security](https://fleetdm.com/handbook/business-operations/application-security) documentation. In addition to this documentation, please refer to [the vendor questionnaires page](https://fleetdm.com/handbook/business-operations/vendor-questionnaires)
+
+### Internal process for handling questionnaires
```

Comment:
@dave let's take this and write it as a responsibility on the sales page.

Reply:
I thought about that, but the process isn't just a sales responsibility, it spans many teams.

Reply:
@dherder It does span a lot of teams but If the AE on the deal is responsible for shepherding the doc through the process. So let's centrally locate the info there.

Reply:
makes sense.